packages/yoastseo/spec/scoring/productPages/seoAssessorSpec.js

```diff
@@ -242,20 +242,34 @@ describe( "running assessments in the product page SEO assessor", function() {
 		] );
 	} );
 
-	it( "additionally runs assessments that require a product with variants or a price", function() {
+	it( "additionally runs assessments that require the SKU to be detectable, and that shouldn't be applicable if the" +
+		"product has variants and we don't want to assess variants", function() {
```

Comment:
The description is hard to read. It is hard to get the purpose of the test. I would suggest something more readable along the lines of:

```suggestion
	it( "runs the productSKUAssessments when applicable (canRetrieveVariantSkus is true) that require the SKU to be detectable, and that shouldn't be applicable if the" +
		"product has variants and we don't want to assess variants", function() {
```